ui/tests/helpers/openapi/expected-secret-attrs.js

```diff
@@ -594,6 +594,14 @@ const pki = {
       label: 'DNS/Email Subject Alternative Names (SANs)',
       type: 'string',
     },
+    certMetadata: {
```

Comment:
`metadata` attr was renamed to `certMetadata` but we don't use either of these in the UI